src/engraving/rw/write/twrite.cpp

```diff
@@ -853,6 +853,28 @@ void TWrite::write(const Breath* item, XmlWriter& xml, WriteContext& ctx)
 
 void TWrite::write(const Chord* item, XmlWriter& xml, WriteContext& ctx)
 {
+    // HACK: foundNotes is a workaround introduced with the "notes in chords" selection filter. A substantial overhaul of our
+    // copy/paste logic would be required to make this fully compatible with de-selected chords - for now we'll simply replace
+    // these chords with a rest of the same duration...
+    bool foundNotes = false;
+    const size_t noteCount = item->notes().size();
+    for (size_t noteIdx = 0; noteIdx < noteCount; ++noteIdx) {
+        if (ctx.canWriteNoteIdx(noteIdx, noteCount)) {
+            foundNotes = true;
+            break;
+        }
+    }
+    if (!foundNotes) {
+        Rest* dummyRest = Factory::createRest(item->segment());
+        dummyRest->setDurationType(item->durationType());
+        dummyRest->setTuplet(item->tuplet());
+        dummyRest->setTicks(item->ticks());
+        dummyRest->setTrack(item->track());
+        write(dummyRest, xml, ctx);
```

Comment:
Something you could try in a next PR: writing a `<location>` tag, to skip ahead by some number of ticks. Seems supported by `Read460::pasteStaff`.